core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1460,10 +1460,8 @@ class ReplicaManager(val config: KafkaConfig,
         warn("Unable to fetch data from remote storage", e)
         return Some(createLogReadResult(e))
     }
-
```

Comment:
Could you please remove those unrelated changes? I guess they are vestige code of tracking this flaky :)

Reply:
sure, just removed it, thanks for the review @chia7712 !